transmission_interface/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ if(BUILD_TESTING)
     test/simple_transmission_loader_test.cpp
   )
   target_include_directories(test_simple_transmission_loader PUBLIC include hardware_interface)
+  target_link_libraries(test_simple_transmission_loader gmock_main)
```

Comment:
is this really needed? It doesn't look familiar. Shouldn't we link using `ament_target_dependencies` and than `ament_cmake_gmock` or something like that?

Reply:
this is how you can avoid defining a main for your test files

Reply:
removed, everythning seem to work still